scripts/qa_crowbarsetup.sh

```diff
@@ -2017,6 +2047,13 @@ function set_proposalvars()
     if iscloudver 5 && [ -z "$want_sles12" ] ; then
         deployceph=
     fi
+    # C4: Cloud6 controller SLE12 node
+    if ! iscloudver 6plus ; then
+        want_sles12_controller=
+    fi
+    if [ -n "$want_sles12_controller" ] && iscloudver 6plus ; then
```

Comment:
you can drop the iscloudver here, because if it was not 6plus, want_sles12_controller will already be unset.